met/docs/Users_Guide/release-notes.rst

```diff
@@ -5,6 +5,15 @@ When applicable, release notes are followed by the GitHub issue number which des
 enhancement, or new feature (`MET GitHub issues <https://github.com/dtcenter/MET/issues>`_).
 Important issues are listed **in bold** for emphasis.
 
+MET Version 10.1.2 release notes (20220516)
+-------------------------------------------
+
+* Bugfixes:
+  
+  * Update static arrays with hard-coded size (`#14 <https://github.com/dtcenter/METplus-Internal/issues/14>`_).
```

Comment:
```suggestion
  * Update static arrays with hard-coded size (`METplus-Internal #14 <https://github.com/dtcenter/METplus-Internal/issues/14>`_).
```
Recommend adding METplus-Internal to avoid confusion from users thinking that "14" refers to the issue number in MET.